packages/prosess-vedtak/src/components/revurdering/VedtakRevurderingForm.jsx

```diff
@@ -123,14 +130,21 @@ export class VedtakRevurderingFormImpl extends Component {
         >
           <VerticalSpacer eightPx />
           <>
-            {visOverstyringKnapp && (
+            {ytelseTypeKode === fagsakYtelseType.FRISINN ? (
+              <VedtakOverstyrendeKnapp
+                toggleCallback={this.onToggleUndertrykkeBrev}
+                readOnly={readOnly || initialValues.skalUndertrykkeBrev === true}
```

Comment:
Her bruker du `initialValues` for å hente ut verdien som du manipulerer i state. Er du kun interessert i den initielle verdien her? Isåfall sliter jeg med å se bruken av det du har på state.

Reply:
Samme kommentar her som over; gjør akkurat det samme som for skalBrukeOverstyrendeFritekstBrev. Men jeg er enig i at dette ikke gir helt mening, kan sjekke litt grundigere om dette faktisk er nødvendig.